synapse/tools/fixes/autoadds.py

```diff
@@ -0,0 +1,190 @@
+import sys
```

Comment:
I was kinda hoping this could be something we did in pure storm.  Anything that would prevent that?

Reply:
We could possibly to that. Building the view tree so we're fixing forked views in order may be a little interesting. Would this then be a pure command then?

Reply:
Probably a lib function ( maybe that could even check a version global and do incremental updates from one place in the future )